evm_loader/program/src/evm/mod.rs

```diff
@@ -81,40 +81,46 @@ macro_rules! trace_end_step {
 pub(crate) use trace_end_step;
 pub(crate) use tracing_event;
 
-#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
+#[derive(Debug, Clone, Eq, PartialEq)]
+#[cfg_attr(not(feature = "library"), derive(Serialize, Deserialize))]
```

Comment:
You introduced a lot of clutter with `cfg_attr(not(feature = "library")` everywhere.
Please remove this where it is not critical.

Following modules can be excluded from the tracer library build:
- entrypoint
- instruction
- account_storage/apply, backend, base

Reply:
I think, it was a good idea to exclude `serde` from BPF build, this can help to reduce BPF program size.

Reply:
The `cfg_attr(not(feature = "library")` removes the code related only to Solana program from the library.

But this introduces a lot of unnecessary clutter without much benefit. The library can have unused code without any performance penalty.

Reply:
@anton-lisanin Done.